components/renderer/CaseStudy.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import LinkWrapper from '@components/LinkWrapper';
+import Image from 'next/image';
+import classNames from 'classnames';
+import { TypePageCaseStudy, TypePageFields } from '@types';
+import * as Contentful from 'contentful';
+
+type CaseStudyPropTypes = {
+    pageCaseStudy: Contentful.Entry<TypePageFields>;
+};
+
+const CaseStudy = ({ pageCaseStudy }: CaseStudyPropTypes) => {
+    const { title } = pageCaseStudy.fields;
+    const { fields: caseStudyPageFields } = pageCaseStudy.fields.content as TypePageCaseStudy;
+    const { summary, heroImage } = caseStudyPageFields.caseStudy.fields;
+    return (
+        <div
+            key={pageCaseStudy.fields.content.sys.id}
+            className="items-center block grid-cols-1 gap-0 mx-6 md:mx-0 md:grid md:grid-cols-14 pb-24"
+        >
+            <div className="col-span-4 col-start-2">
+                <h5
+                    className={classNames(`h5 overline`, {
+                        'text-white': true,
+                    })}
+                >
+                    Case Study
+                </h5>
+                <h3 className={classNames(`h4`, `hover:text-lime`, { 'text-white': true })}>{title}</h3>
```

Comment:
This hover state is on something that isn't clickable. I think the link should wrap the whole block. You can use the tailwind group class to add the hover state: https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state